design/src/App.scss

```diff
@@ -120,6 +120,137 @@ $ribbon-color: #9a0017;
     }
 }
 
+//Form
+.img-overlay {
+  width: 100vw;
+  max-width: 100%;
+  position: absolute;
+  height: 100vh;
+  background-color: rgba(0,0,0,0.7);
+  z-index: 100;
+}
+
+.form-section{
+  width: 100vw;
+  max-width: 100%;
+  position: relative;
+  height: 100vh;
+  background-image: url('../../img/background.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-wrapper{
```

Comment:
Some CSS selectors have a space between the curly brace and the selector name, whereas others don't. A nitty-gritty detail, but important for code base consistency :)

Reply:
Also some values are directly after the colon, whereas others have a space. Consider using a linter, either only in the editor or in the project (so all team members follow the same coding standards).

Reply:
Sure, fully understand. Will change :)